CHANGELOG.md

```diff
@@ -53,6 +53,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v9.0.0-rc2] - 2023-02-02
+
+* (feat) Add [Interchain-Security](https://github.com/cosmos/interchain-security) [v1.0.0-rc4](https://github.com/cosmos/interchain-security/releases/tag/v1.0.0-rc4) provider module. See the [ICS Spec](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/README.md) for more details.
+* (gaia) Bump [ibc-go](https://github.com/cosmos/ibc-go) to [v4.2.0](https://github.com/cosmos/ibc-go/blob/release/v4.2.x/CHANGELOG.md). See [v4.2 Release Notes](https://github.com/cosmos/ibc-go/releases/tag/v4.2.0) for details.
+* (gaia) Bump [packet-forwarding-middleware](https://github.com/strangelove-ventures/packet-forward-middleware) to [v4.0.1](https://github.com/strangelove-ventures/packet-forward-middleware/releases/tag/v4.0.1).
+* (tests) Add [E2E ccv tests](https://github.com/cosmos/gaia/blob/main/tests/e2e/e2e_gov_test.go#L138). Tests covering new functionality introduced by the provider module to add and remove a consumer chain via governance proposal.
+* (tests) Add [integration ccv tests](https://github.com/cosmos/gaia/blob/main/tests/ics/interchain_security_test.go). Imports Interchain-Security's `TestCCVTestSuite` and implements Gaia as the provider chain.
+
```

Comment:
Imports Interchain-Security's `TestCCVTestSuite` and implements  -> 
Import Interchain-Security's `TestCCVTestSuite` and implement